modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -41,3 +44,20 @@ platform:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
 ----
+
+ifdef::vSphere[]
+[IMPORTANT]
+====
+You can configure dual-stack networking in single networking interface only.
```

Comment:
i would direct this question to @cybertron for expert comments.

Reply:
>Are we saying just that "Node custom resource objects will only display the address for the primary address family"?

Yes this is true

Reply:
@rbbratta I went over your comment too quickly...

> OVN can only use a single default gateway interface for both protocols anyway

This is true, but what we are saying here ("configure dual-stack networking in single networking interface only") is not the same as the OVN limitation. We want to put here an even stronger limitation - OVN says you can have as many NICs as you want as long as the default gateway is present only on 1. We want to say that you can have only 1 NIC with addresses configured.

Yeah, I know it sounds like a very strong one, but nevertheless for a full support this is what we want to write. In the next release we will significantly soften it

Reply:
Ok so maybe a bit of clarification, because from the perspective of how we implemented https://issues.redhat.com/browse/OPNET-13 it does not matter at all if you are doing UPI or IPI

The deal here is that I am saying "no matter what and how you deploy, I want you to have only 1 NIC with IPv6 subnet to call your setup supported". And indeed

* https://issues.redhat.com/browse/OPNET-13 is only for IPI (still, on the way we have implemented UPI also but we will not allow to call it "supported")
* we control terraform so VMs created by o/installer have 1 NIC

So in theory you could say that the whole comment about "single NIC with blablabla" is not needed because we can imply it from the IPI-ness. But having seen in the past how customers (ab)use docs and mix&match installation methods, I just feel safer if we embed this statement somehow.

Anyway I leave the final decision to you folks. I just have a feeling that if we don't write it explicitly, sooner or later (rather sooner) we will have someone coming back with this explicit question

Reply:
Okay, thanks for the clarification.  I guess technically Multiple NICs support with OVN plugin in the UPI environment is not supported until https://issues.redhat.com/browse/NP-687 in the next release.

Reply:
@rbbratta @mkowalski So I guess we can choose to keep this note if there are high chances of someone coming with this question. This note would go away with multiple NIC support in the next release. Can one of you confirm if you are fine with this?